README.md

```diff
@@ -35,31 +35,61 @@ vagrant up
 https://dev.futurice.com:8081/vimma/
 
 
-## Running on your machine:
-
-Create a Python3 virtual enviroment and use it for most commands.
+## Manual development setup:
 
 ```bash
+
+wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | apt-key add -
+echo "deb http://dl.google.com/linux/chrome/deb/ stable main " > /etc/apt/sources.list.d/google.list
+
+apt-get update
+apt-get install -y \
+	build-essential vim htop \
+	apache2 libapache2-mod-auth-pubtkt libapache2-mod-wsgi-py3 \
+	rabbitmq-server \
+	supervisor postgresql libpq-dev python-dev \
+	npm git \
+	xvfb chromium-browser chromium-chromedriver firefox \
+	default-jre google-chrome-stable redis-server
+
+ln -s /usr/lib/chromium-browser/chromedriver /usr/local/bin/chromedriver
+
+apt-get install node npm
+ln -s /usr/bin/nodejs /usr/bin/node
+
 virtualenv -p python3 env
 . env/bin/activate
 pip install -r req.txt
 
 npm install
-PATH=`pwd`/node_modules/.bin:$PATH ./scripts/bower-reset.py	# bower install
+cd vimmasite/vimma/static/vimma/components
+bower install
```

Comment:
This and the apt-get commands require everything to be installed globally. I'd leave the manual instructions minimal and local (not global/root). Procboy could have its separate section, like Vagrant has, as they are both alternatives to the manual way.